packages/react-composites/src/composites/CallComposite/components/CapabilitiesChangedNotificationBar.tsx

```diff
@@ -89,13 +89,17 @@ const getCapabilityChangedNotificationString = (
 ): string | undefined => {
   switch (notification.capabilityName) {
     case 'turnVideoOn':
+      /* @conditional-compile-remove(media-access) */
+      return undefined;
```

Comment:
The old notification is being removed because you are using a new notification when audio and video are enabled, right?

Reply:
Yes, that is right.